app/src/org/commcare/utils/FormUploadUtil.java

```diff
@@ -129,6 +129,7 @@ public static FormUploadResult sendInstance(int submissionNumber, File folder,
         if (files.length == 0) {
             Log.e(TAG, "no files to upload");
             listener.cancel(true);
```

Comment:
Doesn't this mean that we're still canceling the `ProcessAndSendTask` when this error is encountered? Is that what we want? (Possible that it is, I just couldn't tell for sure from the discussion).